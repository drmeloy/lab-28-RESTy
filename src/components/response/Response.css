.Response {
  grid-area: response;
  border: 1px solid;
  margin: 10px;
  padding: 10px;
  background: #ffffff4a;
  border-radius: 25px;
  font-family: 'Oswald', sans-serif;
  overflow: auto;
}