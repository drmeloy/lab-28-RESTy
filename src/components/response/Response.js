import React from 'react';
import PropTypes from 'prop-types';
import ResHeaders from './ResHeaders';
import ResResponse from './ResResponse';
import styles from './Response.css';

const Response = ({ responseBody }) => {
  return (
    <div className={styles.Response}>
      <ResHeaders />
      <ResResponse responseBody={responseBody} />
    </div>
  );
};

Response.propTypes = {
  responseBody: PropTypes.object
};

export default Response;
