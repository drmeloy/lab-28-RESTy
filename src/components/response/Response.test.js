import React from 'react';
import { shallow } from 'enzyme';
import Response from './Response';

describe('Response component', () => {
  it('renders Response', () => {
    const wrapper = shallow(<Response />);
    expect(wrapper).toMatchSnapshot();
  });
});
