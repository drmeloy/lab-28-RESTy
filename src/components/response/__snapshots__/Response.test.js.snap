// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Response component renders Response 1`] = `
<div
  className="Response"
>
  <ResHeaders />
  <ResResponse />
</div>
`;
